packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -676,6 +676,11 @@ void main() {
     await gesture.up();
     debugDefaultTargetPlatformOverride = null;
   });
+
+  testWidgets('NestedScrollView with SliverOverlapAbsorber in or out of the first screen', (WidgetTester tester) async {
+    await tester.pumpWidget(const _TestLayoutExtentIsNegative(1));
```

Comment:
Add `// Regression test for https://github.com/flutter/flutter/issues/39963.` above this line.